packages/salesforcedx-vscode-aura/src/views/lwc-node.ts

```diff
@@ -0,0 +1,50 @@
+import { paramCase } from 'change-case';
+import { Command, TreeItem, TreeItemCollapsibleState, Uri } from 'vscode';
+
+import * as path from 'path';
+export enum NodeType {
+    Namespace,
+    Component,
+    WebComponent,
+    Attribute
+}
+function getLabel(label: string, type: NodeType) {
+    if (type === NodeType.WebComponent) {
+        const [ns, ...rest] = label.split(':');
+        return ns + '-' + paramCase(rest.join(''));
+    }
+    return label;
+}
+export class LwcNode extends TreeItem {
+    public children: LwcNode[] = [];
+    constructor(
+        label: string,
+        public readonly tooltip: string,
+        public readonly type: NodeType,
+        public readonly collapsibleState: TreeItemCollapsibleState,
+        public readonly uri?: Uri,
+        public readonly command?: Command
+    ) {
+        super(getLabel(label, type), collapsibleState);
+        this.tooltip = tooltip;
+        this.type = type;
+    }
+
+    get contextValue() {
+        if ((this.type === NodeType.Component || this.type === NodeType.WebComponent) && !this.uri) {
+            return 'external';
+        }
+        return undefined;
+    }
+
+    get iconPath() {
+        if (this.type === NodeType.Namespace) {
+            return path.join(__dirname, '../../../resources/namespace.svg');
```

Comment:
Doesn't this break on windows ? You can use `path.join` to take care of building the uri in correct format depending on the platform being used, e.g.:

```
path.join(__dirname, '..', '..', '..', 'resources', 'namespace.svg');
```

Reply:
Good catch. We haven't tested much on windows - is there a windows machine you guys have access to that I can use?

Reply:
Some of us have personal windows machines, the rest of the team just spins up their own Windows VMs. cc @ntotten

Reply:
I've fixed these and ones I found in the LSPs, just need to test on windows to make sure ternjs in the language server is working correctly